src/services/cohortCreation/fetchCondition.ts

```diff
@@ -86,26 +84,24 @@ type Code = {
   label?: string
 }
 
-const fetchICD9ValueSet = memoize(
+const fetchConditionValueSet = memoize(
   async (): Promise<Code[]> => {
-    const response = await api.get<FHIR_API_Response<IValueSet>>('/ValueSet?url=http://arkhn.com/icd9_VS')
-    const valueSet = getApiResponseResources(response)
-    if (!valueSet || valueSet.length === 0) return []
-    return (
-      valueSet[0]?.compose?.include[0]?.concept
-        ?.map((value) => ({
-          id: value.code,
-          label: value.display
-        }))
-        .sort((a, b) => (a.label && b.label ? a.label.localeCompare(b.label) : 0)) ?? []
-    )
+    const { data: valueSet } = await api.get<IValueSet>(`/ValueSet/$expand?url=${CONDITION_VS_URL}`)
```

Comment:
@nriss a little warning here. It won't work if the `$expand` doesn't or if it's not needed (if the values are in `valueSet.compose.include[x].concept` as before for example).

Reply:
I tried on this vs : ValueSet/$expand?url=http://hl7.org/fhir/ValueSet/ucum-bodylength and got the two defined codes in expansion

I think the problem comes from the VS definition